src/main/java/org/logstash/dissect/fields/Field.java

```diff
@@ -13,17 +13,24 @@
     int NORMAL_ORDINAL_LOWER = 1;
     int APPEND_ORDINAL_BASE = 100;
     int INDIRECT_ORDINAL_HIGHER = 1000;
-    int MISSING_ORDINAL_HIGHEST = 100000;
+    // int MISSING_ORDINAL_HIGHEST = 100000;
     Pattern SUFFIX_REGEX = Pattern.compile("(.+?)(/\\d{1,2}|->|/\\d{1,2}->|->/\\d{1,2})?$");
+    Pattern SUFFIX_ONLY_REGEX = Pattern.compile("^(/\\d{1,2}|->|/\\d{1,2}->|->/\\d{1,2})?$");
```

Comment:
Consider making these Patterns `static`

Reply:
TIL "All the fields declared in an interface are implicitly public, static and final constants"